packages/react-native-windows-init/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "react-native-windows-init",
+  "version": "0.0.8",
```

Comment:
Nit: why 0.0.8?

Reply:
I might have published a few times to do local testing...